packages/@aws-cdk/cfnspec/spec-source/specification/100_sam/100_sam.spec_DeploymentPreference_patch.json

```diff
@@ -30,13 +30,33 @@
               },
               "Hooks": {
                 "Documentation": "https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#deploymentpreference-object",
-                "PrimitiveItemType": "String",
-                "Type": "List",
+                "Type": "Hooks",
```

Comment:
Can we move this to a new patch file, instead of updating the old one?

Reply:
Done!